salt/modules/cp.py

```diff
@@ -8,6 +8,9 @@
 import os
 import logging
 import fnmatch
+import sys
+reload(sys)
+sys.setdefaultencoding('utf-8')
```

Comment:
Well that would need to be gated behind a check for Windows, right? And simply forcing UTF-8 wouldn't account for non-UTF-8 unicode character sets.